laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -417,8 +417,10 @@ public function selectMilestones($id){
         $anesthesiologists = Anesthesiologist::where('updated_at', '>', Carbon::today())
             ->orderBy('last_name')
             ->get();
+            
+        $datas = compact('data1', 'data2', 'data3');
```

Comment:
These datas and data1, data2, data3 variable names seems vague and it unclear without a comment what the data is. Perhaps change it to something else that is more clear like roomChoices, choices, scheduleChoices, or schedulePreferences.

Reply:
I agree that these names are non-descriptive, but in the interest of keeping this PR focused specifically on the blade template, I chose not to change them.

I can expand the scope of this PR to cover refactoring ScheduleDataController if you feel that's appropriate.